### PR TITLE
F23 new admin

### PR DESCRIPTION
**Pull request by chiranthans23**
## Description
Update the officers' page with new officers and moved ex-officers to alumni section

### CODE REVIEW

**Comment by clmurphey** [comment]
@chiranthans23 I think we should update last year's grad comm members (me and spencer 22-23) and Exec comm members (Rohan Das) at the bottom of the officers page where it says alumni. Is that feasible?

**Comment by clmurphey** [comment]
And I noticed that my bio says "second-year PhD student". Can you update that to just say "PhD student"?

File: _data/officers.yml [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-    name: James Luther
-    contact: James.Luther
-    photo: assets/img/officers/james_luther.jpeg
-    pronouns: He/Him/His
-    bio: Jay Luther is a third-year Ph.D. student currently between advisors. He enjoys games of all kinds, reading, and hiking in the front range.
+    name: Corey Lynn Murphey
+    contact: Corey.Murphey@
+    photo: assets/img/officers/corey_murphey.jpg
+    pronouns: She/Her/Hers
+    bio: Corey is a second-year Ph.D. student interested in scientific computing, numerical computation, and computational fluid dynamics. As a classically-trained lyric soprano, Corey loves singing so much that she is trying to incorporate it into her research. In her spare time, she loves playing music, swimming, running, reading, and cuddling with her three Golden Retrievers.
```
Comment (clmurphey) [inline _data/officers.yml:8]: Can we change this to "Corey is a Ph.D. student..."

Comment (chiranthans23) [reply _data/officers.yml]: Done @clmurphey

**Pull request merged**